bin/ycqlsh.py

```diff
@@ -1123,6 +1123,9 @@ def perform_simple_statement(self, statement):
             # CAS INSERT/UPDATE
             self.writeresult("")
             self.print_static_result(result, self.parse_for_update_meta(statement.query_string), time_end)
+        else:
+            if self.timing_enabled and time_end is not None:
```

Comment:
Is there a case when we have `timing_enabled` but do not have `time_end` available? If not, we can skip the check for the latter.

Reply:
I have added second check just as a safeguard. If in case `time_end` is `None` users don't get any errors.